modules/ROOT/pages/available-policies.adoc

```diff
@@ -42,6 +42,9 @@ You can only one policy of each characteristic.
 | Rate Limiting, SLA-Based | Quality of Service | Rate Limited, Client ID required
 | Spike Control | Quality of Service | Throttled, Rate Limited
 | XML Threat Protection | Security | XML Threat Protected
+| Tokenization | Security |
```

Comment:
@nahuel-dallavecchia: Are these _characteristics_ still existing the API Manager UI? If so, what would the characteristics be for the _Tokenization_, _Detokenization_ and _JWT validation_ policies?

Reply:
For those three policies, the characteristic is "Security".